app/editor/src/features/admin/ingests/configurations/AudioClip.tsx

```diff
@@ -20,7 +20,7 @@ export const AudioClip: React.FC = (props) => {
         name="configuration.timeZone"
         tooltip="Timezone of the source"
         options={TimeZones}
-        defaultValue={timeZone}
+        value={timeZone}
```

Comment:
Not sure how `defaultValue` was working before.  Testing today was throwing errors.  Had to fix this on a number of admin forms.